### PR TITLE
added new apiProfiles

### PR DESCRIPTION
**Pull request by bmoore-msft**
Added 2 new apiProfiles and formatted the JSON

### CODE REVIEW

File: schemas/2018-05-01/subscriptionDeploymentTemplate.json [hunk 1]
```diff
@@ -51,14 +53,30 @@
...
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-03-01/Microsoft.Authorization.json#/resourceDefinitions/policyAssignments" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-03-01/Microsoft.Authorization.json#/resourceDefinitions/policyDefinitions" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-03-01/Microsoft.Authorization.json#/resourceDefinitions/policySetDefinitions" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-05-01/Microsoft.Authorization.json#/resourceDefinitions/policyAssignments" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-05-01/Microsoft.Authorization.json#/resourceDefinitions/policyDefinitions" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-05-01/Microsoft.Authorization.json#/resourceDefinitions/policySetDefinitions" }
+                  {
+                    "$ref": "https://schema.management.azure.com/schemas/2017-09-01/Microsoft.Authorization.json#/resourceDefinitions/roleAssignments"
+                  },
```
Comment (TianoMS) [inline schemas/2018-05-01/subscriptionDeploymentTemplate.json]: is this just formatting change?

**Comment by TianoMS** [comment]
>         "2018-06-01-profile"

do we need this change in 2014-04-01-preview/deploymentTemplate.json?

---
Refers to: schemas/2015-01-01/deploymentTemplate.json:17 in b65c9d0. [](commit_id = b65c9d02c51249a531a784f954db46bb46055d0d, deletion_comment = False)

File: schemas/2018-05-01/subscriptionDeploymentTemplate.json (hunk 1 above)
Comment (bmoore-msft) [reply schemas/2018-05-01/subscriptionDeploymentTemplate.json]: yes - everything that's *not* in the apiProfile property is just formatting

**Comment by bmoore-msft** [comment]
the 2014 schema did not have the apiProfile property, I'm not 100% certain it's supported in that version...

**Comment by Tiano2017** [comment]
there are some conflicts..

**Comment by bmoore-msft** [comment]
will open a new PR

**Pull request closed without merging**